src/broker/broker.c

```diff
@@ -432,7 +432,7 @@ int main (int argc, char *argv[])
         log_err ("heaptrace_initialize");
         goto cleanup;
     }
-    if (exec_initialize (ctx.h, ctx.rank, ctx.attrs) < 0) {
+    if (exec_initialize (ctx.h, ctx.overlay, ctx.attrs) < 0) {
```

Comment:
Shouldn't be necessary to change `exec_initialize()` prototype, since authorization doesn't need `struct overlay`?

Reply:
Thanks, good catch. That was left over from the previous attempt. I should have just started over :facepalm: